navigation.yaml

```diff
@@ -297,12 +297,12 @@ kubernetes:
       path: /kubernetes/features
     - title: Managed
       path: /kubernetes/managed
+    - title: Compare
+      path: /kubernetes/compare
     - title: Install
       path: /kubernetes/install
     - title: Docs
       path: /kubernetes/docs
-    - title: Apps
-      path: /managed
```

Comment:
@bethcollins92 Just checking you meant to remove this entry?

Reply:
Yep thanks, this was requested by @alxchalkias